saleor/settings.py

```diff
@@ -236,6 +237,10 @@
 
 LOGIN_REDIRECT_URL = 'home'
 
+INTEGRATIONS_DIR = os.environ.get('INTEGRATIONS_DIR', '')
+INTEGRATIONS_ENABLE_SSL = ast.literal_eval(
+    os.environ.get('INTEGRATIONS_ENABLE_SSL', 'False'))
```

Comment:
Why do we need a specific HTTPS settings for integrations? Either the site uses HTTPS or it does not. And it should if `DEBUG` is disabled.